test/e2e/manifests/apiextensions/composition/functions/setup/composition.yaml

```diff
@@ -36,6 +36,11 @@ spec:
         results:
          - severity: SEVERITY_NORMAL
            message: "I am doing a compose!"
+    credentials:
+      source: Secret
+      secretRef:
+        namespace: crossplane-system
+        name: super-secret
```

Comment:
needs to switch this to the new list format

Reply:
Whoops, serves me right for getting distracted and moving on before the E2E tests finished running. Fixed.